.github/workflows/clang_linux/start.sh

```diff
@@ -4,22 +4,25 @@ set -e
 
 apt-get update -y
 DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-    autoconf automake libtool make cmake ccache pkg-config python3-pip sqlite3 tar zip \
+    autoconf automake libtool make cmake ccache pkg-config python3-pip sqlite3 tar zip curl \
```

Comment:
FYI, added `curl` here (and similar) so that networking is enabled for testing. Otherwise "Skipping network dependent tests" was shown.